packages/navie/src/services/project-info-service.ts

```diff
@@ -20,6 +20,25 @@ export default class ProjectInfoService {
 
     const projectInfo = Array.isArray(response) ? response : [response];
     this.interactionHistory.log('Project info obtained');
+    return projectInfo;
+  }
+
+  promptProjectInfo(isArchitecture: boolean, projectInfo: ProjectInfo[]) {
+    const emitProjectInfo: ProjectInfo[] = projectInfo.map((info) => ({ ...info }));
+
+    const isLargeProject = (appmapStats: AppMapStats) =>
+      appmapStats.packages?.length > 20 ||
+      appmapStats.routes?.length > 20 ||
+      appmapStats.tables?.length > 20;
```

Comment:
These shouldn't be nullish
https://github.com/getappmap/appmap-js/blob/c0b609098e5c52fd4cc193366446faeaa665e305/packages/navie/src/project-info.ts#L10-L15